dev-libs/libpqxx/libpqxx-5.0.1-r1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+inherit python-any-r1
+
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+
+DESCRIPTION="Standard front-end for writing C++ programs that use PostgreSQL"
+SRC_URI="https://github.com/jtv/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="http://pqxx.org/development/libpqxx/"
+LICENSE="BSD"
+SLOT="0"
+IUSE="doc static-libs"
+
+RDEPEND="dev-db/postgresql:="
+DEPEND="${PYTHON_DEPS}
+		${RDEPEND}
+		doc? (
+			 app-doc/doxygen
+			 app-text/xmlto
+		)
+"
+
+src_prepare() {
+	sed -e 's/\[:space:\]/ /g' \
+		-i configure configure.ac configure.ac.in \
+		|| die "Couldn't fix sed syntax"
+
+	sed -e 's/\[\[:space:\]\]/ /g' \
+		-i configure configure.ac configure.ac.in \
+		|| die "Couldn't fix sed syntax"
+
+	sed -e 's/python/python2/' \
+		-i tools/{splitconfig,template2mak.py} \
+		|| die "Couldn't fix Python shebangs"
+
+	eapply_user
```

Comment:
instead of `eapply_user`, call `default`, and call it at the beginning of `src_prepare`